bitnami/mariadb/templates/primary/statefulset.yaml

```diff
@@ -72,6 +72,11 @@ spec:
       {{- else if .Values.priorityClassName }}
       priorityClassName: {{ .Values.priorityClassName | quote }}
       {{- end }}
+      {{- if .Values.primary.runtimeClassName }}
+      runtimeClassName: {{ .Values.primary.runtimeClassName | quote }}
+      {{- else if .Values.runtimeClassName }}
```

Comment:
The value `.Values.runtimeClassName` has not been added to the `values.yaml` file.
Please add the missing value or replace the logic to only use primary/secondary values.

Reply:
It's added, Thanks 👍